ui/src/components/AnswerOption/AnswerOption.css

```
@@ -1,14 +1,19 @@
 .create-poll-field-short {
-    width: 70vw;
+    width: 60%;
     height: 10vh;
     background-color: #f2f2f2;
     border: 1px solid #BDBDBD;
     padding: 0 1vw;
     border-radius: 5px;
     outline: none;
-    margin: 0 0 2vh 10vw;
+    margin: 0 0 7vh 10vw;
+    margin-left: 18.5vw;
+    margin-right: 18.5vw;
```

Pretty sure you can compact the rules here, since `margin` is a shorthand for `margin-*` in any direction. You can do something like:
```suggestion
    margin: 0 18.5vw 7vh 18.5vw;
```
You should get the same functionality.